schemas/iso19139/src/main/plugin/iso19139/index-fields/language-default.xsl

```diff
@@ -709,15 +710,17 @@
     <xsl:for-each select="gmd:referenceSystemInfo/gmd:MD_ReferenceSystem">
         <xsl:for-each select="gmd:referenceSystemIdentifier/gmd:RS_Identifier">
             <xsl:variable name="crs">
-                <xsl:for-each select="gmd:codeSpace/gco:CharacterString/text() | gmd:code/gco:CharacterString/text()">
-                    <xsl:value-of select="."/>
-                    <xsl:if test="not(position() = last())">::</xsl:if>
-                </xsl:for-each>
+            {
+              "code": "<xsl:value-of select="gmd:codeSpace/*/text()"/>:<xsl:value-of select="gmd:code/*/text()"/>",
+              "name": "<xsl:value-of select="gmd:code/*/@xlink:title"/>",
+              "url": "<xsl:value-of select="gmd:code/*/@xlink:href"/>"
+            }
             </xsl:variable>
 
-            <xsl:if test="$crs != ''">
-                <Field name="crs" string="{$crs}" store="true" index="true"/>
```

Comment:
I would continue to have a crs field that we can search on or create a facet on (ie. index=true).
What I would suggest:
* Add the new object in a new field eg. "proj", "crsdetails" ? to be used to display CRS info in record view (to add to dumpfields too in config-lucene.xml)
* Keep the existing field as "crs" and maybe add to this field both CharacterString and Anchor text content